numpy/_array_api/_manipulation_functions.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from ._types import Optional, Tuple, Union, array
+from ._array_object import ndarray
+
+import numpy as np
+
+def concat(arrays: Tuple[array], /, *, axis: Optional[int] = 0) -> array:
```

Comment:
```suggestion
def concat(arrays: Tuple[array, ...], /, *, axis: Optional[int] = 0) -> array:
```

Reply:
Does the array API specify that this must be a tuple?
It seems a bit strict here as any Sequence (or ordered iterable in general) seems sufficient here.

Reply:
That was an explicit decision (see description of https://github.com/data-apis/array-api/pull/42), but it wasn't discussed enough in review. I agree it does seem a bit too strict.